### PR TITLE
feature: added the getLastMembers() function.

### PR DESCRIPTION
**Pull request by argahsuknesib**
Added the getLastMembers() function which returns a list containing the quads, which were last added to the container.

### CODE REVIEW

**Comment by argahsuknesib** [comment]
@woutslabbinck request for review. :)

File: src/ldes/LDESinLDP.ts [hunk 1]
```diff
@@ -341,4 +341,36 @@ export class LDESinLDP implements ILDES {
             }
         }
     }
+
+
+    /**
+     * Returns the last members added to the container based on the timestamp.
+     * @returns {Promise<Quad[]>}
+     */
+    public async getLastMembers(): Promise<Quad[]> {
```
Comment (woutslabbinck) [inline src/ldes/LDESinLDP.ts:350]:
```suggestion
    public async getLastMember(): Promise<Quad[]> {
```
Should it not return the last member based on timestamp?

File: src/ldes/LDESinLDP.ts [hunk 2]
```diff
@@ -341,4 +341,36 @@ export class LDESinLDP implements ILDES {
...
+        const metadata = await this.extractLdesMetadata();
+        const relations = filterRelation(metadata, new Date(0), new Date())
```
Comment (woutslabbinck) [inline src/ldes/LDESinLDP.ts:352]:
```suggestion
        const relations = filterRelation(metadata, new Date(), new Date())
```
Normally this is going to give the most recent fragment, which means the rest of the code will work just as is

File: src/ldes/LDESinLDP.ts [hunk 3]
```diff
@@ -341,4 +341,36 @@ export class LDESinLDP implements ILDES {
...
+                if (lastQuad !== undefined) {
+                    lastQuadList.push(lastQuad)
+                }
+            }
+
+        }
+        return lastQuadList;
+    }
+
 }
```
Comment (woutslabbinck) [inline src/ldes/LDESinLDP.ts:376]: Also add some tests please.

**Review by woutslabbinck — CHANGES_REQUESTED** [review]

**Pull request closed without merging**